mysql-test/suite/galera_nbo/r/galera_nbo_gtid.result

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE t1(i INT PRIMARY KEY);
+#
+# Run ALTER TABLE in NBO mode and verify that it is logged with a proper GTID.
+#
+SET SESSION wsrep_OSU_method="NBO";
+ALTER TABLE t1 ENGINE=InnoDB;
+SET SESSION wsrep_OSU_method=default;
+include/show_binlog_events.inc
+Log_name	Pos	Event_type	Server_id	End_log_pos	Info
+mysqld-bin.000002	#	Gtid	#	#	SET @@SESSION.GTID_NEXT= 'Gtid_set'
```

Comment:
It will always be mysql-bin.000002?

Reply:
Yes. It is always `mysqld-bin.00002` since the server rotates binary logs when node_2 joins the cluster.